src/containers/ProjectAddForm.js

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import { connect}  from 'react-redux'
+import { addNewProject } from '../actions'
+
+const ProjectAddForm = ({dispatch, user}) => {
+  let title, description, beginningDate, dueDate, developer
+  return(
+    <div>
+      <form className="form-inline"
+        onSubmit={e => {
+          e.preventDefault()
+          if (!title.value.trim() || !description.value.trim() || !beginningDate.value.trim() || !dueDate.value.trim() || !developer.value.trim()) {
+            return
+          }
+          const porjectInfo = 'title='+title.value + '&description=' + description.value + '&beginningDate=' + beginningDate.value + '&dueDate='+ dueDate.value + '&developer=' + developer.value
+          dispatch(addNewProject(porjectInfo));
+          title.value = ''
+          description.value = ''
+          beginningDate.value = ''
+          dueDate.value = ''
+          developer.value = ''
+        }}
```

Comment:
For resetting all the input fields of a form please utilise `reset()` method of `HTML DOM`.